docs/src/guides/typo3/deploy/customize.md

```diff
@@ -0,0 +1,117 @@
+
+```bash
+$ composer require pixelant/pxa-lpeh
+```
+
+Second, you will need to add a timeout which will set an HTTP timeout of 3 seconds instead of the default several minutes to a new `public/typo3conf/PlatformshConfiguration.php`. You can see this line in context of the full file in the [configuration](#environment) section below.
+
+```php
+$GLOBALS['TYPO3_CONF_VARS']['HTTP']['timeout'] = 3;
+```
+
+You will still need to enable the `pixelant/pxa-lpeh` extension, which you will be able to do in the [post-install](/guides/typo3/deploy/deploy.md#post-install) steps later in this guide.
+
+### (Optional) Introductory examples
+
+The TYPO3 template for Platform.sh loads a number of example content pages provided by TYPO3 into the site when dependencies are installed pre-build. Their inclusion is completely optional, but a helpful choice if you are creating a new TYPO3 site from scratch using Composer.
+
+```bash
+$ composer require typo3/cms-introduction
+```
+
+## Configuration
+
+You're going to add and/or edit a few files described in the sections below, which will result in the following directory structure:
+
+```bash
+.
+├── config
+│   └── sites
+│       └── main
+│           └── config.yaml
+...
+├── public
+│   └── typo3conf
+│       ├── AdditionalConfiguration.php
+│       └── PlatformshConfiguration.php
+└── src
+    ├── SetupConfiguration.yaml
+    └── SetupDatabase.yaml
+```
+
+### Setup
+
+It's likely that you have a setup configuration file (`src/SetupConfiguration.yaml`) already present in your repository which manages your installation during build when running the `typo3 install:setup` command. On Platform.sh, it is recommended that that file  include at least the following commands.
+
+{{< github repo="platformsh-templates/typo3" file="src/SetupConfiguration.yaml" lang="yaml" >}}
+
+### Site
+
+Similarly, you likely have a `config.yaml` file in your repo, althout it may not be in the `config/sites/main` subdirectory as shown above. This file contains settings for the error handling of your site, the languages it will support, and a few others. For the purposes of this guide, you will need to set the `base` attribute to an environment variable called `PLATFORM_ROUTES_MAIN`. 
+
+{{< github repo="platformsh-templates/typo3" file="config/sites/main/config.yaml" lang="yaml" >}}
+
+You will define this environment variable in the next sections, but it's purpose is to retrieve the root domain (since you have not yet registered a domain name on the Platform.sh project, this will be a hashed placeholder domain generated from the environment) from the environment varable `PLATFORM_ROUTES`.
+
+### Database
+
+Next you will need a `src/SetupDatabase.yaml` file, which is used on the first installation of TYPO3 in the  dedploy hook to set up the database and the initial `admin` user.
+
+{{< github repo="platformsh-templates/typo3" file="src/SetupDatabase.yaml" lang="yaml" >}}
```

Comment:
You can skip this. Instead, explain how to push an existing DB's content to Platform.sh.

The PlatformshConfiguration.php file should set all of the DB credentials automatically, so if that one is included correctly, it no more configuration should be necessary.